lib/spack/spack/enums.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+"""Enumerations used throughout Spack"""
+import enum
+
+
+class DBStatus(enum.Flag):
```

Comment:
This one was renamed, cause we have 3 different `class InstallStatus` in `develop` right now.

Reply:
It's not a status of the DB but of the record, and we never use an abbreviation like "DB" elsewhere.

Reply:
I can use `InstallRecordStatus`. Point is, I would like to avoid duplicate class names.

Reply:
Done in https://github.com/spack/spack/pull/47321/commits/3008094c674f53a933b7d02f4b9ed0280ea0cbc1